src/transactions/index.js

```diff
@@ -16,7 +16,9 @@ import transfer from './0_transfer';
 import registerSecondPassphrase from './1_register_second_passphrase';
 import registerDelegate from './2_register_delegate';
 import castVotes from './3_cast_votes';
-import registerMultisignature from './4_register_multisignature_account';
+import registerMultisignature, {
+	signMultisignature,
```

Comment:
Does this not belong more in `utils` alongside signing transactions normally? `4_register_multisignature_account` exports a transaction creator.

Reply:
I think having in the top layer make more sense rather than having in `utils` because imo this creating signature object is the same layer as creating transaction.

Reply:
I think it might even better to have it on separate file

Reply:
Agree with top level, but definitely a separate file I would say.